src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -1861,6 +1861,18 @@ public function testSignal()
         $this->assertTrue($command->signaled);
         $this->assertTrue($dispatcherCalled);
     }
+
+    public function testSignalableCommandInterfaceWithoutSignals()
+    {
+        $command = new SignableCommand();
```

Comment:
This test is not relevant if `pcntl_signal` is available, let's skip it
```suggestion
        if (SignalRegistry::isSupported()) {
            $this->markTestSkipped('This test is not relevant if pcntl_signal() is enabled.');
        }
        
        $command = new SignableCommand();
```

Reply:
this test is actually relevant all the time. It should still pass with pcntl available (it was already passing without the patch in that case, but that does not make the test irrelevant)

Reply:
sure, it's just testing something that is already tested when pcntl is available. Nvm